frontend/src/components/Status.js

```diff
@@ -308,6 +308,7 @@ const Status = ({
 								<Info
 									label="Broker Traffic"
 									infoIcon
+									infoId={"broker-traffic-info-icon"}
```

Comment:
Rename `infoId` to just `id`. (also the following occurrences.

Reply:
Done, see latest commit.